.github/workflows/build-test-cxx.yml

```diff
@@ -0,0 +1,140 @@
+﻿name: 'Build, Test, Package ITK Remote Module'
+
+on:
+  workflow_call:
+    inputs:
+      cmake-options:
+        required: false
+        type: string
+      itk-cmake-options:
+        required: false
+        type: string
+      itk-git-tag:
+        required: false
+        type: string
+        default: '171fb2ba33a87041f99328a2f26612ff33aa9cc8'
+
+jobs:
+  build-test-cxx:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-11]
+        include:
+          - os: ubuntu-20.04
+            c-compiler: "gcc"
+            cxx-compiler: "g++"
+            cmake-build-type: "MinSizeRel"
+          - os: windows-2019
+            c-compiler: "cl.exe"
+            cxx-compiler: "cl.exe"
+            cmake-build-type: "Release"
+          - os: macos-11
+            c-compiler: "clang"
+            cxx-compiler: "clang++"
+            cmake-build-type: "MinSizeRel"
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
Also here https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3rc04/Testing/ContinuousIntegration/AzurePipelinesLinuxPython.yml#L29

Reply:
Submitted https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/issues/4 to track